<launch>
  <node pkg="obst_avoid" name="ImageProcesser" type="ImageProcesser" respawn="true" output="screen"/>
</launch>
